spec/rails_helper.rb

```diff
@@ -10,6 +10,9 @@
 require 'shoulda/matchers'
 require 'factory_bot_rails'
 require 'ffaker'
+require 'simplecov'
+SimpleCov.start
+puts "required simplecov"
```

Comment:
Does that mean coverage is generated on every request?

Reply:
@abitrolly now yes. We able to on/off this by ENV variable. 
And it is open question start always with tests or by request(only on CI for example).

Reply:
Does that submit how users are browsing the web site to 3rd party service?

Reply:
@abitrolly no.